atomic_reactor/plugins/post_rpmqa.py

```diff
@@ -5,9 +5,19 @@
 This software may be modified and distributed under the terms
 of the BSD license. See the LICENSE file for details.
 """
+import os
+import subprocess
+import tempfile
+import tarfile
 
 from atomic_reactor.plugin import PostBuildPlugin
 from atomic_reactor.utils.rpm import rpm_qf_args, parse_rpm_output
+from atomic_reactor.plugins.pre_reactor_config import get_list_rpms_from_scratch
+from docker.errors import APIError
+
+RPMDB_PATH = '/var/lib/rpm'
```

Comment:
Can we get this path by calling `rpm.expandMacro` instead of hardcoding it?

Reply:
I'd rather keep explicit path here, so we don't get any surprises